tests/test_validate_outputs.py

```diff
@@ -413,16 +418,16 @@ def test_forced_phot_diffim(self):
         # No external calibrations are applied to the diffim forced
         # measurements, so no tables should be produced the detectors with
         # astrometry failures.
+        num_expected_dia_successes = (self._num_exposures_good_templates
+                                      + self._num_surprise_diffim_successes
+                                      - self._num_forced_astrom_failures)
+        # forced source counts depend on detector/tract overlap.
         self.check_sources(
             ["forced_diff", "forced_diff_diaObject"],
-            self._num_exposures_good_templates - self._num_forced_astrom_failures,
-            self._min_sources
+            num_expected_dia_successes,
```

Comment:
This isn't a problem with the code, but is there any reason you made the `num_expected_dia_successes` variable here in the `test_forced_phot_diffim` function and passed that into `check_sources`, but didn't do the same in the` test_image_difference` or `test_forced_source_tables` functions? Of course the result's the same either way; just wondering about it from a consistency and ease of future bulk editing standpoint.

Reply:
Because this function used this value twice, and the other two used it once.  Assigning a variable when you use it twice reduces lines of code/duplication.   Assigning a variable when you use it once adds lines of code. That it is all.